packages/yoga/src/Dialog/web/Header.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Text from '../../Text';
 
 const Header = props => (
-  <Text.H4 bold as="header" ta="center" mb="large" {...props} />
+  <Text.H4 bold as="h2" ta="center" mb="large" tabIndex={-1} {...props} />
```

Comment:
I don't think tabindex="-1" is necessary in this situation

Reply:
The recommendation for the initial focus of a Dialog varies according to the context. In our case, we are always focusing on the first element of the dialog, which would be the title in most cases.

To add the initial focus to a non-interactive element, it is recommended to use tabindex="-1".

https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/
![Screenshot 2024-11-19 at 08 21 20](https://github.com/user-attachments/assets/3e7181ec-ac49-41ce-ab7e-29112c9b9b14)